enhancements/network/baremetal-ipi-network-configuration.md

```diff
@@ -0,0 +1,275 @@
+---
+title: baremetal-ipi-network-configuration
+authors:
+- "@cybertron"
+- "@hardys"
+- "@zaneb"
+reviewers:
+- "@kirankt"
+- "@dtantsur"
+- "@zaneb"
+approvers:
+- "@trozet"
+creation-date: 2021-05-21
+last-updated: 2021-07-07
+status: provisional|implementable
+
+see-also:
+- "/enhancements/host-network-configuration.md"
+- "/enhancements/machine-config/mco-network-configuration.md"
+- "/enhancements/machine-config/rhcos/static-networking-enhancements.md"
+---
+
+# Baremetal IPI Network Configuration
+
+Describe user-facing API for day-1 network customizations in the IPI workflow,
+with particular focus on baremetal where such configuration is a common
+requirement.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
```

Comment:
We could probably mark the status as implementable and check both these I think, given the pretty thorough discussion so far?